triangle/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "20170429" %}
+
+package:
+  name: triangle
```

Comment:
We need to name this `python-triangle` to avoid classing with the c lib `triangle`.
Ideally we should also make this one depend on the triangle recipe but `setup.py` would need a major patch to do that :-/